mapie/classification.py

```diff
@@ -43,29 +43,30 @@ class MapieClassifier(BaseEstimator, ClassifierMixin):
         Method to choose for prediction interval estimates.
         Choose among:
 
-        - "naive", sum of the probabilities until the 1-alpha thresold.
+        - ``"naive"``, sum of the probabilities until the 1-alpha thresold.
 
-        - "score", based on the the scores
+        - ``"label"`` (formerly called ``"score"``), based on the the scores
```

Comment:
maybe add that "label" (and now LAC : Least Ambiguous set-valued Classifier) stands for